hawkular-alerts-bus/src/main/java/org/hawkular/alerts/bus/listener/AvailDataListener.java

```diff
@@ -115,7 +115,7 @@ protected void onBasicMessage(AvailDataMessage msg) {
     protected String convertReceivedMessageClassNameToDesiredMessageClassName(String className) {
 
         if (className.equals("org.hawkular.metrics.component.publish.AvailDataMessage")) {
-            return "org.hawkular.alerts.bus.messages.AvailDataMessage";
+            return "org.hawkular.alerts.bus.api.AvailDataMessage";
```

Comment:
I like the idea of getCanonicalName() but will leave as-is in order to get this in asap.  It can be changed in passing as part of future work.